egs/fisher_swbd/s5/local/chain/run_tdnn_lstm_1a_svd.sh

```diff
@@ -0,0 +1,379 @@
+#!/bin/bash
+
+# The script is organized as below.
+# First we train the baseline LSTMP-TDNN config chain model for few epochs on the multien data,
+# Then, we perform SVD based refactoring of all the Affine components in this baseline final.mdl,
+# in order to reduce the overall model parameters size,
+# as determined by the bottleneck dim value or Energy and Shrinkage threshold values.
+# Then, we finetune the weight parameters of the refactored model using entire multien data for single epoch.
+
+# Commands used for performing  WER decoding on a testset (like rt03) :
```

Comment:
It shouldn't be necessary to run commands not in the script in order to get the WER.  The whole point of these scripts is that you just run them.  So you should add something to the end of the script with more decodings, with a later stage. 
You should be using compare_wer.sh to compare the output, and including appropriate baselines in there (e.g. the pre-SVD version of the same system).  And I still don't like the name with _svd in it; should be _1b.